semanticcpg/src/main/scala/io/shiftleft/semanticcpg/passes/cfgdominator/CfgDominator.scala

```diff
@@ -87,10 +89,10 @@ class CfgDominator[NodeType](adapter: CfgAdapter[NodeType]) {
     finger1
   }
 
-  private def createPostOrderNumbering(cfgEntry: NodeType): Map[NodeType, Int] = {
+  private def createPostOrderNumbering(cfgEntry: NodeType): mutable.Map[NodeType, Int] = {
     var stack = (cfgEntry, adapter.successors(cfgEntry).iterator) :: Nil
-    var visited = Set.empty[NodeType]
-    var numbering = Map.empty[NodeType, Int]
+    var visited = mutable.Set.empty[NodeType]
```

Comment:
If we're touching this code anyways, may as well `val visited = ...`

Reply:
Done.